src/components/HistoryTables/MsigProposals/ProposalRow.tsx

```diff
@@ -45,11 +44,11 @@ export default function ProposalHistoryRow(props: ProposalHistoryRowProps) {
         </Badge>
       </TD>
       <TD>
-        <AddressWOptionalLink
-          onClick={e => e.stopPropagation()}
-          address={proposal.approved[0].robust || proposal.approved[0].id}
-          addressHref={addressHref}
-          inspectingAddress={inspectingAddress}
+        <AddressLink
+          id={proposal.approved[0].robust ? '' : proposal.approved[0].id}
```

Comment:
That depends what we want to display in the table rows.

The behaviour previously was:
If `address.robust` exists, display that, otherwise, display the ID. That is what this code is doing right now.

If we pass both the `id` and `address` prop to AddressLink, it will show like this:
`f17uoq ... twpryy
 (t07633)`

Since we have limited space in the table, I imagine we just one to show one or the other.